scripts/quick-install-ubuntu.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail  ## https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+
+# This script assumes root permissions on a fresh Ubuntu Digital Ocean droplet
+# with these recommended specs: 16 GB Memory / 8 Intel vCPUs / 320 GB Disk
+
+# dismiss the popups
+export DEBIAN_FRONTEND=noninteractive 
+
+apt update -y && apt upgrade -y
+
+apt install docker jq -y
+
+## need to do this for Linux <-> docker compose when not installed with Docker Desktop
+mkdir -p ~/.docker/cli-plugins
+curl -SL https://github.com/docker/compose/releases/download/v2.11.2/docker-compose-linux-x86_64 -o ~/.docker/cli-plugins/docker-compose
+chmod +x ~/.docker/cli-plugins/docker-compose
+
+# install latest `laconic-so`
+mkdir -p  ~/bin
+curl -L -o ~/bin/laconic-so https://github.com/cerc-io/stack-orchestrator/releases/latest/download/laconic-so
+chmod +x ~/bin/laconic-so
+
+# set here for this script
+export PATH=$PATH:~/bin
```

Comment:
`export` not needed here?

Reply:
Scratching my head about that. I don't think it is. `PATH` is valid in the script, without the `export`.

Reply:
ah, you're right. brain fart.